modules/cudf/src/column.ts

```diff
@@ -700,6 +701,98 @@ export interface Column<T extends DataType = any> {
    */
   isNotNaN(memoryResource?: MemoryResource): Column<Bool8>;
 
+  /**
+   * Creates a column of `BOOL8` elements indicating strings in which all characters are valid for
+   * conversion to floats.
+   *
+   * @param memoryResource The optional MemoryResource used to allocate the result Column's device
+   *   memory.
+   * @returns A non-nullable column of `BOOL8` elements with `true` representing convertible
+   *   values
+   */
+  isFloat(memoryResource?: MemoryResource): Column<Bool8>;
+
+  /**
+   * Returns a new strings column converting the float values from the provided column into strings.
+   *
+   * Any null entries will result in corresponding null entries in the output column.
+   *
+   * For each float, a string is created in base-10 decimal. Negative numbers will include a '-'
+   * prefix. Numbers producing more than 10 significant digits will produce a string that includes
+   * scientific notation (e.g. "-1.78e+15").
+   *
+   * @param memoryResource The optional MemoryResource used to allocate the result Column's device
+   *   memory.
+   *
+   *  @returns A string Column with floats as strings.
+   */
+  fromFloats(memoryResource?: MemoryResource): Column<Utf8String>;
```

Comment:
Should this method be inside of `FloatSeries`?

Reply:
No these are all methods that go on the low-level `Column`, because they call out to `libcudf` directly. This is similar to the way `containsRe`  is on `Column` even though it only applies specifically to *string* columns. I agree that the wide API on column is not perfect in some ways, but it seems to be an acceptable compromise given that it is not user-facing.